apps/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -247,7 +249,8 @@ export class RushCommandLineParser extends CommandLineParser {
 
             enableParallelism: command.enableParallelism,
             ignoreMissingScript: command.ignoreMissingScript || false,
-            ignoreDependencyOrder: command.ignoreDependencyOrder || false
+            ignoreDependencyOrder: command.ignoreDependencyOrder || false,
+            doNotFailOnWarnings: !!command.allowWarningsInSuccessfulBuild
```

Comment:
Is there any difference between `doNotFailOnWarnings` vs `allowWarningsInSuccessfulBuild`?  If not, let's use a consistent name everywhere